src/transformation/rgbz.c

```diff
@@ -556,9 +798,28 @@ k4a_buffer_result_t transformation_depth_image_to_color_camera_internal(
 
     context.depth_image = transformation_init_input_image(depth_image_descriptor, depth_image_data);
 
+    context.custom_image = transformation_init_input_image(custom_image_descriptor, custom_image_data);
+
     context.transformed_image = transformation_init_output_image(transformed_depth_image_descriptor,
                                                                  transformed_depth_image_data);
 
+    context.transformed_custom_image = transformation_init_output_image(transformed_custom_image_descriptor,
+                                                                        transformed_custom_image_data);
+
+    context.enable_custom8 = false;
+    context.enable_custom16 = false;
+    if (custom_image_descriptor->format == K4A_IMAGE_FORMAT_MONO8)
+    {
+        context.enable_custom8 = true;
+    }
+    else if (custom_image_descriptor->format == K4A_IMAGE_FORMAT_MONO16)
+    {
+        context.enable_custom16 = true;
+    }
+
+    context.interpolation_type = interpolation_type;
+    context.invalid_value = (uint16_t)(invalid_custom_value & 0xffff);
+
     if (K4A_FAILED(TRACE_CALL(transformation_depth_to_color(&context))))
```

Comment:
>transformation_depth_to_color [](start = 30, length = 29)

is this function should be named with the depth+custom similarly as other places?

Reply:
I didn't rename the internal functions since there is only one implementation of depth_to_color that handles both cases; depth only and depth + custom.